Assignment2/src/serviceUtil/userService.java

```diff
@@ -0,0 +1,110 @@
+package serviceUtil;
+
+import model.Student;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.List;
+import java.util.Scanner;
+import util.*;
+import static serviceUtil.inputOutputUtil.inputRnoToBeDeleted;
+import util.fileReadWriteUtil;
+
+public class userService {
+    private static Student student;
+    private static String studentRecFileName;
+    private static List<Student> studentRecordList;
+
+    /*
+       public static void addUser() {
+           studentRecFileName = fileReadWriteUtil.getFile(); // extract and read student file from memory
+           studentRecordList = fileReadWriteUtil.readFile("studentData.txt");
+           student = inputOutputUtil.getInputUserDetails(); // get command line input of user details
+           studentRecordList.add(student); // add all the new entries to a list
+       }
+     */
+    public static void deleteUser() {
+        studentRecFileName = "/Users/amulverma/Downloads/Assignment2/studentData.txt";
+        studentRecordList = fileReadWriteUtil.readFile(studentRecFileName);
+        int deleteRno = inputOutputUtil.inputRnoToBeDeleted();
+        boolean deleted = false;
+        for (int i = 0; i < studentRecordList.size(); i++) {
+            if (studentRecordList.get(i).getRollNumber() == deleteRno) { // check for matching roll number in the
+                // records
+                studentRecordList.remove(i);
+                saveUserDetails(studentRecordList, studentRecFileName);
+                deleted = true;
+                System.out.print("Data successfully deleted!");
+                break;
+            }
+        }
+        if (!deleted) {
+            System.out.println("There is no entry in student record with given roll number!");
```

Comment:
If fileReadWriteUtil.readFile(studentRecFileName); returns with error, you will be showing the wrong message to the user.